authors/urls.py

```diff
@@ -33,7 +35,11 @@
    permission_classes=(permissions.AllowAny,),
 )
 
+router = DefaultRouter()
+router.register("api/articles/mytags", ArticleTagViewSet)
```

Comment:
Hi @llwasampijja , any particular reason as to why are we using router now

Reply:
@nadralia  yes, I am attempting to take full advantage of the Django rest framework, which allows me to automatically generate the different URLs for the endpoints basing on the modelviewset!
In other words, instead of having to list each endpoint for this application, the router does the heavy lifting for us!